### PR TITLE
Fix #1323: Removed cp classes causing links to have backgrounds

### LINKED ISSUE #1323: Cp-Misc Background on Links
Go to Administration > Other > Language

image

These are likely meant to be buttons or just harmless links, but they are colored out due to the cp-misc class attached to them.

### PR DESCRIPTION
**Pull request by jonathanzhang53**
Removed cp-misc classes that were causing the backgrounds to appear. These cp-misc classes seem to be merely adding background color and color to the text, but it doesn't work out with links.

Fixes #1323

### CODE REVIEW

**Comment by robbyoconnor** [comment]
Fixed your commits history -- again please use `git pull --rebase`

**Comment by robbyoconnor** [comment]
The title for this is horrible -- please give a description of what you changed and why.

File: interface/language/language.php [hunk 1]
```diff
@@ -35,10 +35,10 @@
 <table>
  <tr>
   <td class="small" colspan='4'>
-   <a href="?m=definition" class='cp-positive' onclick="top.restoreSession()"><?php echo htmlspecialchars(xl('Edit Definitions'),ENT_NOQUOTES); ?></a> | 
-   <a href="?m=language" class='cp-positive' onclick="top.restoreSession()"><?php echo htmlspecialchars(xl('Add Language'),ENT_NOQUOTES); ?></a> | 
-   <a href="?m=constant" class='cp-positive' onclick="top.restoreSession()"><?php echo htmlspecialchars(xl('Add Constant'),ENT_NOQUOTES); ?></a> |
-   <a href="?m=manage" class='cp-misc' onclick="top.restoreSession()"><?php echo htmlspecialchars(xl('Manage Translations'),ENT_NOQUOTES); ?></a>
```
[the remark this answers is not in the document]
Comment (muarachmann) [reply interface/language/language.php]: Is this link an issue or a PR

[the remark this answers is not in the document]
Comment (muarachmann) [reply interface/language/language.php]: @jonathanzhang53 it redirects me here

[the remark this answers is not in the document]
Comment (jonathanzhang53) [reply interface/language/language.php]: Sorry about that. I linked the wrong one. #1323

File: interface/language/language.php [hunk 2]
```diff
@@ -61,6 +61,6 @@
 endswitch;
 ?>
 
-<BR><A HREF="lang.info.html" class='cp-misc' TARGET="_blank"><?php echo htmlspecialchars(xl('Info'),ENT_NOQUOTES); ?></A>
+<BR><A HREF="lang.info.html" TARGET="_blank"><?php echo htmlspecialchars(xl('Info'),ENT_NOQUOTES); ?></A>
```
Comment (muarachmann) [inline interface/language/language.php]: Stripping off these is a baaaaadddd idea. I know but this is not what we intended for when placing this cause we didn't finish this prior to gsoc. I guess team alpha will be coming on board any time soon to re-tackle this. IMO let them be cause we aim at making stuff uniform once gci is over

Comment (muarachmann) [reply interface/language/language.php]: Also there will be a removal of themes from the codebase and we will just be left with 2 themes.

Comment (jonathanzhang53) [reply interface/language/language.php]: @muarachmann this is in style.php so it applies to all themes. Is there really nothing that can be done for this specifically?

Comment (muarachmann) [reply interface/language/language.php]: What i am saying is we will do a batch revamp of the buttons across the whole EHR at once. You are welcome to join if interested.And for that we will need those `cp-xxxx` classes. We started but got shut cause of  gsoc

Comment (jonathanzhang53) [reply interface/language/language.php]: @muarachmann I see. closing this then. I will definitely see if I have time to contribute.

Comment (muarachmann) [reply interface/language/language.php]: Thanks @jonathanzhang53 we will appreciate your help. You are doing great so far i like your debug skills though 👍

**Pull request closed without merging**